ftw/news/upgrades/20200527193316_remove_plone_app_referenceablebehavior/types/ftw.news.NewsFolder.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<object name="ftw.news.NewsFolder"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="ftw.news" >
+
+    <property name="title" i18n:translate="">News Folder</property>
+    <property name="description" i18n:translate="">A container for news items.</property>
+    <property name="icon_expr"></property>
+    <property name="allow_discussion">False</property>
+    <property name="global_allow">True</property>
+    <property name="filter_content_types">True</property>
+    <property name="allowed_content_types">
+        <element value="ftw.news.News"></element>
+        <element value="ftw.news.NewsListingBlock" />
+        <element value="ftw.simplelayout.FileListingBlock" />
+        <element value="ftw.simplelayout.GalleryBlock" />
+        <element value="ftw.simplelayout.MapBlock" />
+        <element value="ftw.simplelayout.TextBlock" />
+        <element value="ftw.simplelayout.VideoBlock" />
+    </property>
+
+    <property name="schema">ftw.news.contents.news_folder.INewsFolderSchema</property>
+    <property name="klass">ftw.news.contents.news_folder.NewsFolder</property>
+    <property name="add_permission">ftw.news.AddNewsFolder</property>
+
+    <property name="behaviors">
+        <element value="plone.app.dexterity.behaviors.metadata.IBasic"/>
+        <element value="plone.app.content.interfaces.INameFromTitle" />
+        <element value="plone.app.dexterity.behaviors.exclfromnav.IExcludeFromNavigation" />
+        <element value="ftw.simplelayout.interfaces.ISimplelayout" />
+    </property>
+
+    <property name="immediate_view">simplelayout-view</property>
+    <property name="default_view">simplelayout-view</property>
+    <property name="view_methods">
+        <element value="simplelayout-view"/>
+        <element value="news_listing"/>
+    </property>
+    <property name="default_view_fallback">False</property>
+
+    <alias from="(Default)" to="(dynamic view)" />
+    <alias from="view" to="(selected layout)" />
+    <alias from="edit" to="@@edit" />
+    <alias from="sharing" to="@@sharing" />
+
+    <action
+        action_id="view"
+        title="View"
+        category="object"
+        condition_expr=""
+        url_expr="string:${object_url}"
+        visible="True">
+        <permission value="View"/>
+    </action>
+
+    <action
+        action_id="edit"
+        title="Edit"
+        category="object"
+        condition_expr=""
+        url_expr="string:${object_url}/edit"
+        visible="True">
+        <permission value="Modify portal content"/>
+    </action>
+
+</object>
```

Comment:
You only have to add the delta otherwise it will overwrite policy level customization. 
```suggestion
<?xml version="1.0"?>
<object name="ftw.news.NewsFolder">
  <property name="behaviors" purge="False">
    <element value="plone.app.referenceablebehavior.referenceable.IReferenceable" remove="True"/>
  </property>
</object>
```